ble2mqtt/protocols/govee.py

```diff
@@ -0,0 +1,31 @@
+"""
+Decoder for Govee H5074 temperature/humidity sensor.
+Based on https://github.com/wcbonner/GoveeBTTempLogger/blob/master/goveebttemplogger.cpp (MIT License)
+"""
+from __future__ import annotations
+
+import math
+import struct
+
+
+class GoveeDecoder:
+    def __init__(self, raw_data: bytes) -> None:
+        if len(raw_data) != 7:
+            raise ValueError("Govee data must be 7 bytes long")
+        self.data: tuple[int, ...] = struct.unpack("<xhhBx", raw_data)
```

Comment:
Maybe this annotation syntax is not compatible with python 3.7. I'm trying to keep it compatible with older python versions to be ably to run it on OpenWrt starting from 19.07

Reply:
The same syntax is used here: https://github.com/devbis/ble2mqtt/blob/main/ble2mqtt/protocols/ruuvi.py#L18

Is there something I should use instead?

Reply:
Maybe: I just copied the code in ruuvi.py verbatim from another source.  (Style fixes were subsequently applied.)